lib/hpke/include/hpke/userinfo_vc.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#include <memory>
+#include <optional>
+
+#include <bytes/bytes.h>
+#include <chrono>
+#include <hpke/signature.h>
+#include <map>
+
+using namespace bytes_ns;
+
+namespace hpke {
+
+struct UserInfoVC
+{
+private:
+  struct ParsedCredential;
+  std::shared_ptr<ParsedCredential> parsed_cred;
+
+public:
+  explicit UserInfoVC(std::string jwt);
```

Comment:
why not ``` std::string& jwt ```

Reply:
`jwt` is always copied into `raw`.  Passing by value makes this explicit.